### ISSUE TITLE
Support For Categories

### ISSUE THREAD

**Issue by sprankhub** [avstudnitz/AvS_ScopeHint2#26]
Thanks a lot for this nice tool!

One nice addition would be to show the same hint for categories as well (on the category edit page).

**Comment by avstudnitz** [comment]
@sprankhub You are right. Please see https://github.com/avstudnitz/AvS_ScopeHint2/pull/5 for a WIP version of that.

**Comment by sprankhub** [comment]
If I am not mistaken, #5 is a different thing. It adds support for the categories a **product** is assigned to, so this works on the product edit page.

What I had in mind was to have the hint on category edit pages for category attributes.

**Comment by thedotwriter** [comment]
Just did what you asked for, go give it a review ; )
https://github.com/avstudnitz/AvS_ScopeHint2/pull/27
